components/reusable-components/login-form/LoginForm.tsx

```diff
@@ -81,40 +87,22 @@ const LoginForm: React.FC<LoginFormProps> = ({
             Заборавена лозинка?
           </Link>
         </div>
-        <button type="submit" className={styles.loginBtn} disabled={isLoading}>
-          {isLoading ? 'Најавување...' : 'Најави се'}
+        <button type="submit" className={styles.loginBtn} disabled={loginMutation.isLoading}>
+          {loginMutation.isLoading ? 'Најавување...' : 'Најави се'}
         </button>
+        {loginMutation.error && (
+          <div className={styles.errorMessageContainer}>
+            <Image src={error} alt="Linkedin" />
+            <p className={styles.errorMessage}>Погрешни креденцијали. Обиди се повторно</p>
+          </div>
+        )}
         <Turnstile
           sitekey={process.env.NEXT_PUBLIC_TURNSTILE || ''}
           onVerify={(token) => setTurnstileToken(token)}
           theme={isLightTheme ? 'light' : 'dark'}
           size="invisible"
         />
       </form>
-      <div className={styles.loginSocials}>
-        <p>или продолжи со</p>
-        <div className={styles.socialIcons}>
-          <Link href="https://github.com/learnhubmk" target="_blank" rel="noopener noreferrer">
-            <Image className={styles.socialIcon} src={github} alt="Github" />
-          </Link>
-          <Link href="/" target="_blank" rel="noopener noreferrer">
-            <Image className={styles.socialIcon} src={google} alt="Youtube" />
-          </Link>
-          <Link
-            href="https://www.linkedin.com/company/102600044/admin/feed/posts/"
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            <Image className={styles.socialIcon} src={linkedin} alt="Linkedin" />
-          </Link>
-        </div>
-        <p>
-          Немаш креиран профил?{' '}
-          <Link href="/signup" className={styles.signup}>
-            Регистрирај се
-          </Link>
-        </p>
-      </div>
```

Comment:
why is this part of the component deleted? This is the reusable login component that will be used all over the site: admin-panel, content-panel and platform. And it is done as per the Figma Design

Reply:
I have reworked the LoginForm.tsx and now it accepts only one prop and that is onSubmit function.
I changed the previous one because i found it anti-next.js pattern to make the login page.tsx file a client component.
I prefer ( and also as a best practice ) to keep the page.tsx files server components.

Since, no hooks are allowed in a server component, i created two files that serves as containers where i import the necessary hooks and send prop values to LoginForm.tsx ( path: components/reusable-components/login-form ). Then, i import those components in the page.tsx files (one for content-panel and another for admin-panel), therefore keeping the file server component.